src/reaction/views.py

```diff
@@ -26,13 +57,12 @@ def post(self, request):
             "text": <>,
         }
         """
-        profile = Profile.objects.get(user=request.user)
         data = json.loads(request.body)
         episode_id = data.get("episode_id")
         if not EpisodeDetail.objects.filter(id=episode_id).exists():
             return failure_response(f"Episode with id {episode_id} does not exist!")
         text = data.get("text")
-        reaction = Reaction(episode=EpisodeDetail.objects.get(id=episode_id), text=text, author=profile)
+        reaction = Reaction(episode=EpisodeDetail.objects.get(id=episode_id), text=text, author=request.user.profile)
```

Comment:
lol i was being dumb in my previous pr but nobody caught it so i am fixing it here haha